pgmpy/estimators/expert.py

```diff
@@ -62,6 +62,7 @@ def estimate(
         orientations=set([]),
         use_cache=True,
         show_progress=True,
+        expert_knowledge: ExpertKnowledge = None,
```

Comment:
I think this is a more important argument compared to use_cache and show_progress, as it affects the behaviour of the algorithm, so I would suggest putting this above `use_cache`.